test/providers/ipc_cuda_prov_consumer.c

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2024 Intel Corporation
+ *
+ * Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+
+#include <umf/pools/pool_disjoint.h>
+#include <umf/providers/provider_cuda.h>
+
+#include "cuda_helpers.h"
+#include "ipc_common.h"
+#include "ipc_cuda_prov_common.h"
+
+int main(int argc, char *argv[]) {
+    if (argc < 2) {
+        fprintf(stderr, "usage: %s <port> [shm_name]\n", argv[0]);
```

Comment:
`[shm_name]` - not needed

Reply:
Done. Also, I fixed a similar issue in the L0 IPC test.